Extensions/PrimitiveDrawing/Extension.cpp

```diff
@@ -477,6 +558,53 @@ void DeclarePrimitiveDrawingExtension(gd::PlatformExtension& extension) {
       .SetFunctionName("GetOutlineOpacity")
       .SetIncludeFile("PrimitiveDrawing/ShapePainterObject.h");
 
+  obj.AddExpression(
+        "OutlineOpacity",
+        _("Outline opacity"),
+        _("Check if the opacity of the outline (a number from 0 to 1, 1 being fully opaque)."),
+        "",
+        "res/conditions/opacity.png")
+      .AddParameter("object", _("Shape Painter object"), "Drawer")
+      .SetFunctionName("GetOutlineOpacity")
+      .SetIncludeFile("PrimitiveDrawing/ShapePainterObject.h");
+
+  obj.AddAction("UseAbsoluteCoordinates",
+                _("Use absolute coordinates"),
+                _("Set the coordinates of the Shape Painter to be absolute"),
+                _("Change the coordinate for _PARAM0_ to absolute"),
+                _("Setup"),
+                "res/actions/position24.png",
+                "res/actions/position.png")
+      .AddParameter("object", _("Shape Painter object"), "Drawer")
+      .SetFunctionName("SetCoordinatesAbsolute")
+      .SetIncludeFile("PrimitiveDrawing/ShapePainterObject.h");
+
+  obj.AddAction("UseCoordinatesRelative",
```

Comment:
UseRelativeCoordinates

Remember: the adjective is almost always *before* the noun in English.